### ISSUE TITLE
[Broken Website] Khronos Vulkan Documentation

### ISSUE THREAD

**Issue by ShalokShalom** [darkreader/darkreader#3461]
**Website**
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#ANativeWindow

**How to Reproduce**
An example of this is
> 1. Go to x site
> 2. The whole page does not load

**Expected behavior**
The page loads.

**Actual behavior**
The page doesnt load.

**Screenshots**
Not necessary.

**System info:**
 - OS: Android
 - Browser: Firefox
 - Darkreader Version: current
 
**Additional context**
Happens on both mobile and desktop Firefox.

**Comment by Gusted** [comment]
Hi @ShalokShalom!

Do you have dynamic mode enabled?
It took around 30 seconds with slow 4g(currently on a island) on a oneplus 7 pro

Regards
Gusted

**Comment by AFaurholt** [comment]
@Gusted 
Yeah I'm running with dynamic mode and I have the same issue in desktop Firefox

**Comment by ShalokShalom** [comment]
Yes, the default dynamic mode and it stuck on my home wifi.

**Comment by ShalokShalom** [comment]
Update: Firefox just updated its Android version and introduced the new renderer that is based on Rust.

This one works just fine, so the issue might be only reproducible on the Desktop and maybe iOS.

**Comment by Gusted** [comment]
Ah, Firefox nightly?

**Comment by ShalokShalom** [comment]
![Screenshot_2020-08-18-15-55-31-53](https://user-images.githubusercontent.com/6344099/90521882-54485c00-e16b-11ea-864f-ba07c1bc9c85.jpg)

**Comment by ShalokShalom** [comment]
My normal Firefox just got all the new features😊

**Comment by Gusted** [comment]
Unfortunate no NL yet ![](https://i.imgur.com/B7TPSiu.png) Maybe I can get some sideload from apkmirror. I will have a look soon If I don't forget.
